fuel/app/themes/default/partials/widget/detail.php

```diff
@@ -1,52 +1,88 @@
-					<div class="tooltip" style="display: {{ feature.show ? 'inline-block' : 'none' }}">{{ feature.description }}</div>
-				</dd>
+			<div id="pics-scroller-container">
+				<div id="pics-scroller">
+					<div ng-class="{playing: !showDemoCover, loading: demoLoading}" ng-style="{'min-height': demoHeight, width: demoWidth}">
+						<img ng-src="{{widget.screenshots[0].full}}" ng-show="showDemoCover" ondragstart="return false">
+						<div id="demo-cover" ng-class="{hidden: !showDemoCover, loading: demoLoading}" ng-style="{'background-image': demoScreenshot}">
+							<button class="green" ng-click="showDemoClicked()">
+								<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M8 5v14l11-7z"/><path d="M0 0h24v24H0z" fill="none"/></svg>
+								Play a demo now!
+							</button>
+							<div id="demo-cover-background"></div>
+						</div>
+						<div id="player-container" ng-if="!showDemoCover">
+							<section class="widget" ng-controller="playerCtrl" ng-class="{ preview: isPreview }">
+								<header ng-if="isPreview" class="preview-bar"></header>
+								<div class="center" ng-show="type == 'flash' || type == 'html'">
+									<iframe ng-attr-src="{{ htmlPath }}" ng-if="type == 'html'" id="container" class="html" scrolling="yes" fullscreen-dir></iframe>
+									<div id="container" ng-if="type =='flash'"></div>
+								</div>
+								<div id="container" ng-if="type =='noflash'">
+									<?= Theme::instance()->view('partials/noflash') ?>
+								</div>
+							</section>
+						</div>
+						<h3>{{!showDemoCover ? 'Playing ' : '' }}Demo</h3>
+					</div>
+
+					<div ng-repeat="screenshot in widget.screenshots">
+						<img ng-src="{{screenshot.full}}">
+						<div class="screenshot-drag-cover"></div>
+						<h3>Screenshot {{$index + 1}} of {{numScreenshots}}</h3>
+					</div>
+				</div>
 			</div>
-			<dt ng-show='widget.supported_data.length'>Supported Data:</dt>
-			<div class="aligner">
-				<dd ng-repeat='data in widget.supported_data'>
-					<a class="supported_data" ng-mouseover="data.show=true" ng-mouseout="data.show = false">{{ data.text }}</a>
-					<div class="tooltip" style="display: {{ data.show ? 'inline-block' : 'none' }}">{{ data.description }}</div>
-				</dd>
+
+			<div>
+				<button class="demo-dot" ng-class="{selected: selectedImage == 0}" ng-click="selectImage(0)">Demo</button>
+				<button class="pic-dot" ng-repeat="s in widget.screenshots" ng-class="{selected: selectedImage == $index + 1}" ng-click="selectImage($index + 1)"></button>
 			</div>
-		</dl>
-		<section class="right widget_right_selection">
-			<h4>Want to see it in action?</h4>
-			<p>
-				<a id="demoLink" class="action_button green circle_button" href='{{ widget.demourl }}' target="_blank">
-					<span class="arrow arrow_right"></span>
-					Play a demo now!
-				</a>
-			</p>
+		</div>
+
+		<section class="bottom">
+			<dl id="metaData" class="inline_def">
```

Comment:
definition lists should be used a little differently semantically.
Will need to do some work to reformat the tags and get styles to work again - 
Should be:

```
<div>title</div>
<dl>
  <dt></dt>
  <dd></dd>
  ... repeat
</dl>
```